tests/utils/test_metric_utils.py

```diff
@@ -0,0 +1,61 @@
+"""Unit tests for metrics utilities.
+
+Author: Travis Driver
+"""
+
+import unittest
+
+import trimesh
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler, Cal3Bundler
+from gtsfm.common.keypoints import Keypoints
+
+import gtsfm.utils.metrics as metric_utils
+
+
+class TestMetricUtils(unittest.TestCase):
+    """Class containing all unit tests for metric utils."""
+
+    def test_mesh_inlier_correspondences(self) -> None:
+        """Tests `compute_keypoint_intersections()` function."""
+        # Create cube mesh with side length one centered at origin.
+        box = trimesh.primitives.Box()
+
+        # Create arrangement of two camera pointing at center of one of the cube's faces.
+        calibration = Cal3Bundler(10, 0, 0, 1, 1)
+        target_pos = [0.5, 0, 0]
+        cam_i1 = PinholeCameraCal3Bundler().Lookat([2, 1, 0], target_pos, [0, -1, 0], calibration)
+        cam_i2 = PinholeCameraCal3Bundler().Lookat([2, -1, 0], target_pos, [0, -1, 0], calibration)
```

Comment:
prefer a named arg for (0, -1, 0) and for [2, -1, 0] to help the reader understand what it represents

Reply:
fixed